### PR TITLE
Improve refresh_all script to ignore false positives and remove false positives from expected json files

### PR DESCRIPTION
**Pull request by Mindavi**

### CODE REVIEW

File: tests/lacrosse/tx29-it_tx35dth-it/gfile002-TX35DTH-IT.json [hunk 1]
```diff
@@ -1,2 +1 @@
-{"time" : "@0.127256s", "brand" : "LaCrosse", "model" : "LaCrosse-TX29IT", "id" : 10, "battery_ok" : 1, "newbattery" : 1, "temperature_C" : 23.800, "mic" : "CRC"}
```
Comment (Mindavi) [inline tests/lacrosse/tx29-it_tx35dth-it/gfile002-TX35DTH-IT.json:1]: It seems like both signals are in this file, so maybe it would be good to split it up instead of removing this expected output.

Comment (zuckschwerdt) [reply tests/lacrosse/tx29-it_tx35dth-it/gfile002-TX35DTH-IT.json:1]: Always good to have "bad" files as tests to make sure corner cases work. But yeah, this output won't fit into any test logic, likely best to split the cu8.

Comment (Mindavi) [reply tests/lacrosse/tx29-it_tx35dth-it/gfile002-TX35DTH-IT.json]: I've just removed the updated file for now, but it would be good to handle this better in the future.
